java/jdart-regression/URLDecoder01/Main.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Contributed to SV-COMP by Falk Howar
+ * License: MIT (see /java/jdart-regression/LICENSE-MIT)
+ *    
+ */
+
+import org.sosy_lab.sv_benchmarks.Verifier;
+
+public class Main {
+
+  public static void main(String[] args) {
+
+    String s1 = "some-url+" + Verifier.nondetString();
+
+    try {
+      String s2 = java.net.URLDecoder.decode(s1, "UTF-8");
+      if ( !s2.startsWith("some") ) {
+        assert false;
+      }
+    } catch (java.io.UnsupportedEncodingException e) {
+      // assert sth. in anoter benchmark
```

Comment:
```suggestion
      // assert sth. in another benchmark
```

Reply:
Not sure what this comment means. This catch block is unreachable as far as I can see (and is just needed because of the checked exception thrown by decode).

Reply:
You are right. I removed the comment